InfiniteBlanK3T.Calculator/Program.cs

```diff
@@ -0,0 +1,169 @@
+            int attempts = 0;
+            // Display title as the C# console calculator app.
+            Console.WriteLine("Console Calculator in C#\r");
+            Console.WriteLine("------------------------\n");
+
+            while (!endApp)
+            {
+                // Declare variables
+                List<double> numInputs = new List<double> { };                
+                string numInput = "";
+                double result = 0;                
+                int userDigit;                
+                string userInput;
+                double cleanNum = 0;
+
+                // Ask the user to choose an operator.
+                if (cal.CountResultsList() > 0)
+                {
+                    Console.WriteLine("Your list of past results: " + cal.ResultsList);
+
+                    Console.Write("Do you want to clear it? (y/n): ");
+
+                    userInput = Console.ReadLine();
+
+                    if (userInput == "y")
+                    {
+                        cal.ClearResult();
+                    }
+                }
+
+                Console.Write("How many digits you want to enter?: ");                
+
+                userInput = Console.ReadLine();
+
+                while (!int.TryParse(userInput, out userDigit))
+                {
+                    Console.Write("Invalid input. Please try again:");
+                    userInput = Console.ReadLine();
+                }
+                if (userDigit > 1)
+                {
+                    Console.WriteLine("\nChoose an option from the following list:");
+                    Console.WriteLine("\ta - Add");
+                    Console.WriteLine("\ts - Subtract");
+                    Console.WriteLine("\tm - Multiply");
+                    Console.WriteLine("\tp - To the power of (only for 2 digits)");
+                    Console.WriteLine("\td - Divide");
+                }
+                else
+                {
+                    Console.WriteLine("Choose an option from the following list:");
+                    Console.WriteLine("\ts - Square root");
+                    Console.WriteLine("\tp - Power of two");
+                    Console.WriteLine("\tsin - Trigonometry Sin()");
+                    Console.WriteLine("\tcos - Trigonometry Cos()");
+                    Console.WriteLine("\ttan - Trigonometry Tan()");
+                }
+                Console.Write("Your option? ");
+
+
+                string menu = Console.ReadLine();
+
+                while (menu == null || userDigit > 2 && menu == "p")
+                {
+                    Console.Write("Invalid option picked. Please try again:");
+                    menu = Console.ReadLine();
+                }
+
+                if (numInputs.Count > 0)
+                {
+                    userDigit -= 1;
+
+                }
+                if (userDigit == 1)
+                {
+                    // Ask the user to type the first number.
```

Comment:
In general avoid commenting, unless it's absolutely necessary. Use those to explain to other people (or your future self) things that can't be figure out in the code itself. The less you rely on comments, the better you name your members, and vice-versa.